blueocean-core-js/src/js/services/PipelineService.js

```diff
@@ -25,27 +25,16 @@ export class PipelineService extends BunkerService {
     }
 
     /**
-     * Gets pager for /blue/pipelines
-     *
-     * @returns {Pager}
-     */
-    allPipelinesPager() {
```

Comment:
Once again, this file is in core-js, but is only used by pager - why is it in core-js?